app/views/schools/contacts/index.html.erb

```diff
@@ -1,32 +1,36 @@
 <% content_for :page_title do %>
-  <%= @school.name %> Alert Contacts
+  <%= t('schools.contacts.index.title', school_name: @school.name) %>
 <% end %>
 
-<h1 property="name"><%= link_to @school.name, @school %> Alert Contacts</h1>
+<h1 property="name">
+  <%= link_to @school.name, @school %> <%= t('schools.contacts.index.alert_contacts') %>
+</h1>
 
 <%= render 'shared/contact_explainer' %>
```

Comment:
Text extracted for this in a seperate pr https://github.com/Energy-Sparks/energy-sparks/pull/2261